code/modules/mob/living/carbon/human/species_types/grod.dm

```diff
@@ -0,0 +1,396 @@
+	if(chem.type == /datum/reagent/toxin/pestkiller || chem.type == /datum/reagent/toxin/plantbgone)
+		H.adjustToxLoss(3)
+		H.reagents.remove_reagent(chem.type, chem.metabolization_rate)
+		return FALSE
+	return ..()
+/datum/action/innate/grod/crownspider
+	name = "Detach Crown"
+	desc = "Detach your crown from your current body. This should only be done in dire circumstances!"
+	icon_icon = 'icons/mob/actions/actions_grod.dmi'
+	button_icon_state = "crownspider"
+
+/datum/action/innate/grod/crownspider/Grant()
+	..()
+	if(!isgrod(owner))
+		return
+
+/datum/action/innate/grod/crownspider/Activate()
+	if(!isgrod(owner)) //Stop trying to break shit
+		return
+	var/mob/living/carbon/human/H = owner
+	if(!istype(H.getorganslot(ORGAN_SLOT_BRAIN), /obj/item/organ/brain/grod))
+		to_chat(H, "<span class = 'warning'>You dont have a crown! Contact a coder!</span>")
+		return
+	if(H.incapacitated())
+		to_chat(H, "<span class='warning'>You cannot use this ability right now</span>")
+		return
+	if(alert("Are you sure you wish to leave your current body, this will cause massive damage to your Crown!",,"Yes", "No") == "No")
+		return
+	do_leave_body()
+
+/datum/action/innate/grod/crownspider/proc/do_leave_body() //disconnect from body
+	var/mob/living/carbon/human/H = owner
+	var/datum/mind/M = H.mind
+	var/list/organs = H.getorganszone(BODY_ZONE_HEAD, 1)
+	var/turf = get_turf(H)
+
+	for(var/obj/item/organ/brain/I in organs)
+		I.Remove(H, 1)
+
+	var/mob/living/simple_animal/hostile/crown_spider/crown = new(turf)
+
+	crown.name = "[H.name]'s Crown"
+	crown.color = "#" + H.dna.features["mcolor"]
+	for(var/obj/item/organ/brain/I in organs)
+		I.forceMove(crown)
+		crown.health = crown.health*abs(1-(I.damage/I.maxHealth)) //Adjust crown-mob's health to match brain health
+
+	crown.origin = M
+	if(crown.origin)
+		crown.origin.active = 1
+		crown.origin.transfer_to(crown)
+		to_chat(crown, "<span class='warning'>Your consiousness returns to its Crown and you leave your body!</span>")
+
+/datum/species/grod/get_custom_icons(part)
+	switch(part)
+		if("head")
+			return 'icons/mob/species/grod/onmob_grod_head.dmi'
+		if("mask")
+			return 'icons/mob/species/grod/onmob_grod_mask.dmi'
+		if("gloves")
+			return 'icons/mob/species/grod/onmob_grod_gloves.dmi'
+		if("uniform")
+			return 'icons/mob/species/grod/onmob_grod_under.dmi'
+		if("ears")
+			return 'icons/mob/species/grod/onmob_grod_ears.dmi'
+		if("back")
+			return 'icons/mob/species/grod/onmob_grod_back.dmi'
+		if("shoes")
+			return 'icons/mob/species/grod/onmob_grod_shoes.dmi'
+		if("glasses")
+			return 'icons/mob/species/grod/onmob_grod_glasses.dmi'
+		if("neck")
+			return 'icons/mob/species/grod/onmob_grod_neck.dmi'
+		if("generic")
+			return 'icons/mob/species/grod/onmob_grod_generic.dmi'
+		if("belt")
+			return 'icons/mob/species/grod/onmob_grod_belt.dmi'
+		/*if("bloodmask") NOT READY
+			return 'icons/mob/species/grod/bloodmask_grod.dmi'*/
```

Comment:
I saw code referencing bloodmask lower down, I think this needs to be implemented? bloodying is kind of important